packages/react-server/src/ReactFizzServer.js

```diff
@@ -1553,12 +1552,28 @@ function abortTask(task: Task): void {
 
     if (!boundary.forceClientRender) {
       boundary.forceClientRender = true;
-      const error = new Error(
-        'This Suspense boundary was aborted by the server.',
-      );
+      let error =
+        reason === undefined
+          ? // eslint-disable-next-line react-internal/prod-error-codes
+            new Error('signal is aborted without reason')
```

Comment:
Actually, on a second thought. This is kind of a weird error message in the Node API because there was no signal involved.